integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -666,6 +668,21 @@ protected void checkDataFileExists(Path dir) {
     }
   }
 
+  protected void checkTableLocation(Path dir) {
+    Assertions.assertTrue(dir.toString().equals(hdfs.getUri() + LOCATION));
+  }
+
+  protected void deleteDirIfExists() {
```

Comment:
`deleteDirIfExists(String path)` is more general, try to avoid using global variables, it's unmaintainable.